packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -112,6 +114,7 @@ const int _kObscureShowLatestCharCursorTicks = 3;
 ///    with a [TextEditingController].
 ///  * [EditableText], which is a raw region of editable text that can be
 ///    controlled with a [TextEditingController].
+///  * <https://flutter.dev/docs/cookbook/forms/text-field-changes#2-use-a-texteditingcontroller>
```

Comment:
Linking to a cookbook example is a good idea. I suggest making it a bit more readable:
```suggestion
///  * Learn how to use a [TextEditingController] in one of our [cookbook recipe]s.(https://flutter.dev/docs/cookbook/forms/text-field-changes#2-use-a-texteditingcontroller)
```

Reply:
@Piinks Thank you for reviewing. Already make that changes.